doc/content/devices/adding-devices/_index.md

```diff
@@ -55,63 +59,105 @@ See [ID and EUI constraints]({{< ref "reference/id-eui-constraints" >}}) for gui
 
-The **Network Server**, **Application Server**, and **Join Server** addresses should correctly point to the address of {{% tts %}} deployment you are using.
+If you need to use a method of activation other than OTAA, create a multicast group, specify a Class B or Class C device, or change Rx Delay and Frequency plan settings, see the [Advanced settings](#advanced-settings) section below. Otherwise, proceed to [OTAA devices](#otaa-devices).
 
-Click the **Start** button to proceed to the Basic Settings page.
+### OTAA Devices
 
-{{< figure src="manual-settings.png" alt="Register manually" >}}
+Over-the-Air-Activation (OTAA) is the secure, scalable way to activate LoRaWAN devices. All commercially available LoRaWAN devices support OTAA, and it is selected by default. If you are using a custom or DIY device, and cannot use OTAA, see the [Activation by Personalisation](#abp-devices) section.
 
 {{< note >}}
-This guide covers adding a device using [OTAA]({{< ref "reference/glossary#over-the-air-activation" >}}) (the most secure and preferred activation method) and [LoRaWAN version]({{< ref "reference/glossary#lorawan-version" >}}) MAC V1.0.2 (the most common LoRaWAN version, although newer versions are better and more secure). Names and keys may vary slightly for other versions.
+The example in this guide covers adding a device using [OTAA]({{< ref "reference/glossary#over-the-air-activation" >}}) (the most secure and preferred activation method) and [LoRaWAN version]({{< ref "reference/glossary#lorawan-version" >}}) MAC V1.0.2 (the most common LoRaWAN version, although newer versions are better and more secure). Names and keys may vary slightly for other versions, but the process is the same and any differences are noted.
 {{</ note >}}
 
+Enter your **DevEUI**. This should be provided by your manufacturer for commercial devices. If your device is programmable, you may generate an EUI using the **Generate** button, and program it in your device.
+
+Enter a **JoinEUI/AppEUI** if provided by your manufacturer. If your device is programmable, you may use the **Fill with zeros** button, and then program the same JoinEUI/AppEUI (`0000000000000000`) in the device.
+
+If your manufacturer provides an **AppKey**, enter it. Otherwise, use the generate button to create one, and program it in to your device.
+
 Give your device a unique **End device ID**.
 
 {{< note >}}
 See [ID and EUI constraints]({{< ref "reference/id-eui-constraints" >}}) for guidelines about choosing a unique ID.
 {{</ note >}}
 
-Enter a **JoinEUI/AppEUI** if provided by your manufacturer. If your device is programmable, you may use all 0's, and then program the same JoinEUI/AppEUI in the device.
+{{< figure src="manual-network-settings-otaa.png" alt="Manually create OTAA end device" >}}
 
-Enter your **DevEUI**.
+Click **Register end device** to create the end device.
 
-**Name** and **Description** are optional fields to help you identify your device.
+### Advanced Settings
```

Comment:
I would also move this section above OTAA. Having this section between OTAA and ABP makes an impression like advanced settings are something that you choose after you fill in the OTAA information, which is wrong. If people are using ABP, they will probably just skip this, read the ABP section and wonder why some fields are missing in the Console. I would move this up and say something like "in case you are using an ABP device, you want to change the device class to other than class A, you want to set RX1 and RX2 settings manually, or you want to use external Join Server, expand the advanced activation, LoRaWAN class and cluster settings" or something like that